lychee.toml

```diff
@@ -16,4 +16,6 @@ exclude = [
     "^https://desktop.version.rancher.io/v1/checkupgrade",
     "^https://github.com/k3s-io/k3s/releases/download",
     "^https://storage.googleapis.com/kubernetes-release/release",
```

Comment:
I don't think this is still relevant (my impression is that Kubernetes moved off Google infra a while back), but that's not relevant for this PR.

Reply:
> Kubernetes moved off Google infra a while back

They did, but our older docs (up to 1.15.0) still reference the old location. So we exclude the link here because we don't want to update the old docs for this.

And yes, we handle this inconsistently; sometimes we do fix URLs in the old docs.